docs/modules/ROOT/pages/governance.adoc

```diff
@@ -200,11 +200,12 @@ The Governor will automatically detect the clock mode used by the token and adap
 // SPDX-License-Identifier: MIT
```

Comment:
Note: We have to move this example code to a separate file like the others in this article.